galleries/users_explain/figure/backends.rst

```diff
@@ -248,3 +248,87 @@ backend, use ``module://name.of.the.backend`` as the backend name, e.g.
 ``matplotlib.use('module://name.of.the.backend')``.
 
 Information for backend implementers is available at :ref:`writing_backend_interface`.
+
+.. _figures_not_showing:
+
+Debugging the figure windows not showing
+----------------------------------------
+
+Sometimes things do not work as expected, usually during an install.
+
+If you are using a Notebook or integrated development environment (see :ref:`notebooks-and-ides`),
+please consult their documentation for debugging figures not working in their
+environments.
+
+If you are using one of Matplotlib's graphics backends (see :ref:`standalone-scripts-and-interactive-use`), make sure you know which
+one is being used:
+
+.. code-block:: python3
+
+   import matplotlib
+
+   print(matplotlib.get_backend())
+
+Try a simple plot to see if the GUI opens:
+
+.. code-block:: python3
+
+   import matplotlib
+   import matplotlib.pyplot as plt
+
+   print(matplotlib.get_backend())
+   plt.plot((1, 4, 6))
+   plt.show()
+
+If it does not, you perhaps have an installation problem.  A good step at this
+point is to ensure that your GUI toolkit is installed properly, taking
+Matplotlib out of the testing.  Almost all GUI toolkits have a small test
+program that can be run to test basic functionality.  If this test fails, try re-installing.
+
+QtAgg, QtCairo, Qt5Agg, and Qt5Cairo
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Test ``PyQt5``:
+
+If you have ``PySide`` or ``PyQt6`` installed rather than ``PyQt5``, just change the import
+accordingly:
```

Comment:
Doubled colon in the paragraphs.

Reply:
Sorry, not sure what you mean here....

Reply:
Both paragraphs end in a colon, implying that something should come after, but there's nothing supposed to be between these two paragraphs. Perhaps just leave out the first paragraph?

Reply:
Oh duh, I see now.  Thanks,

Reply:
... fixed (finally)